composer.json

```diff
@@ -17,23 +17,23 @@
         }
     ],
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": "^7.1",
         "doctrine/orm": "^2.4.5",
         "sonata-project/admin-bundle": "^3.1",
         "sonata-project/core-bundle": "^3.0",
         "sonata-project/exporter": "^1.3.1",
-        "symfony/console": "^2.8 || ^3.0",
-        "symfony/doctrine-bridge": "^2.8 || ^3.0",
-        "symfony/form": "^2.8 || ^3.0",
-        "symfony/framework-bundle": "^2.8 || ^3.0",
-        "symfony/security": "^2.8 || ^3.0",
-        "symfony/security-acl": "^2.8 || ^3.0"
+        "symfony/console": "^2.8 || ^3.2",
+        "symfony/doctrine-bridge": "^2.8 || ^3.2",
+        "symfony/form": "^2.8 || ^3.2",
+        "symfony/framework-bundle": "^2.8 || ^3.2",
+        "symfony/security": "^2.8 || ^3.2",
+        "symfony/security-acl": "^2.8 || ^3.2"
```

Comment:
security-acl must be max 3.0 (3.2 does not exists)

Reply:
Thanks for the tip